lib/location/cloud_service/cloud_service_nrf.c

```diff
@@ -8,18 +8,23 @@
 #include <zephyr/logging/log.h>
 #if defined(CONFIG_NRF_CLOUD_MQTT)
 #include <net/nrf_cloud_location.h>
-#else
+#elif defined(CONFIG_NRF_CLOUD_REST)
 #include <net/nrf_cloud_rest.h>
+#else
```

Comment:
Can this be `elif defined(CONFIG_NRF_CLOUD_COAP)` ?

It's not worth blocking the PR if that's not practical, but it seems wise to be explicit. If we push forward on this, can that change be in a followup PR?